integration/.graphqlrc.yml

```diff
@@ -15,10 +15,3 @@ extensions:
       subCreate: true
```

Comment:
With that in mind we can also have crud config and location in js

Reply:
To be fair, the file can be removed all together now. Let me do it.

Reply:
Done.